2_adventures/001/chitankensuru/06_other_studying/README.md

```diff
@@ -0,0 +1,4 @@
+##Studying
+
+Will be busy today with learning about process mining and R. So coding will
```

Comment:
Just following an introductory R course through [Coursera] that's part of the Data Science specialization. All new for me. No magic going on. And a handy book review you wrote. Thanks

[Coursera]:https://www.coursera.org/course/rprog